VKappProject/VKappProject/View/ShadowView.swift

```diff
@@ -0,0 +1,42 @@
+// ShadowView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Вью с тенью
+final class ShadowView: UIView {
+    // MARK: - Private Properties
+
+    @IBInspectable private var shadowRadius: CGFloat = 8 {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+
+    @IBInspectable private var shadowOpacity: Float = 0.5 {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+
+    @IBInspectable private var shadowColor: UIColor = .black {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+
+    // MARK: - Public Properties
+
+    override class var layerClass: AnyClass {
+        CAShapeLayer.self
+    }
```

Comment:
```suggestion
    // MARK: - Public Properties

    override class var layerClass: AnyClass {
        CAShapeLayer.self
    }
```
Перенести перед Private Properties